server/models/userProfile.js

```diff
@@ -0,0 +1,62 @@
+const mongoose = require('mongoose')
+
+const userProfile = new mongoose.Schema({
+    firstName: {
+        type: 'string',
+        required: false
+    },
+    lastName: {
+        type: 'string',
+        required: false
+    },
+    gender: {
+        type: 'string',
+        required: false
+    },
+    birthDay: {
+        type: Date,
+        default: null
+    },
+    email: {
+        type: 'string',
+        required: false
+    },
+    phoneNumber: {
+        type: 'string',
+        required: false
+    },
+    address: {
+        type: 'string',
+        required: false
+    },
+    city: {
+        type: 'string',
+        required: false
+    },
+    country: {
+        type: 'string',
+        required: false
+    },
+    description: {
+        type: 'string',
+        required: false
+    },
+    available: { type: 'boolean', required: true },
+    availableDays: [{
```

Comment:
Based on this design for availability, wouldn't it be inconvenient for the user to select multiple individual dates?

Instead they can have availability for days of the week (Monday Tuesday and so on..) between a  start date and end date (2021/09/21 - 2021/12/31)